### DISCUSSION TITLE
Subtitles

### DISCUSSION THREAD

**Discussion by gowth2m** [nt4f04uNd/sweyer#16] [Ideas]
How about add lyrics feature. Not sure there is an api but local lyrics.

**Comment by nt4f04uNd** [comment]
I personally find this not really useful in the mobile music player so this feature is on the bottom of my todo list

If I implement it, I will probably make it as a button that opens genius.com page. I haven't seen a free music lyrics API out there and adding lyrics manually is definitely not what I want to have in the app, as this seems to be useless - you will have it only on that device and it requires too many actions to just paste the lyrics that you could easily find online

If you have other ways how you would see it feel free to share them
